pwndbg/gdblib/kernel/kallsyms.py

```diff
@@ -112,7 +122,7 @@ def find_token_table(self) -> int:
             if len(ascii_candidates) == 1:
                 candidates = ascii_candidates
             elif len(candidates) == 0:
-                print(M.error("No candidates for token_table"))
+                print(M.error("No candidates for token_table, maybe uncompressed kallsyms"))
```

Comment:
Not sure if we want this error print here at this point?
1) Should probably be a better message if we keep it
2) We print info message in init that we detected compressed vs uncompressed and attempt to proceed with that?

Reply:
Changed it to info message for both cases